agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/statsbeat/Feature.java

```diff
@@ -46,12 +46,13 @@
   // JAVA_HTTP_CLIENT_DISABLED(16), no longer used
   // JAXWS_DISABLED(17), no longer used
   RABBITMQ_DISABLED(18),
-  SPRING_INTEGRATION_DISABLED(19),
+  SPRING_INTEGRATION_DISABLED(
+      19), // preview instrumentation, spring-integration is ON by default in OTEL
   LEGACY_PROPAGATION_DISABLED(20),
-  GRIZZLY_DISABLED(21), // preview instrumentation
+  GRIZZLY_ENABLED(21), // preview instrumentation, grizzly is OFF by default in OTEL
```

Comment:
@heyams just a heads up, I'm hoping this changes, and grizzly instrumentation can be on by default in the future, not sure if it changes your thoughts on tracking or not though

Reply:
that's fine.. when that happens, i can rename to disabled then.